.github/workflows/publish-release.yml

```diff
@@ -45,19 +45,9 @@ jobs:
         run: |
           ./gradlew release -Prelease.useAutomaticVersion=true
           ./gradlew -Pversion=${VERSION} publish -x test -PpublishToCentral=true
-      - name: Create Github release from the release tag
-        run: |
-            curl --request POST 'https://api.github.com/repos/ballerina-platform/module-ballerinax-java.jdbc/releases' \
-            --header 'Accept: application/vnd.github.v3+json' \
-            --header 'Authorization: Bearer ${{ secrets.BALLERINA_BOT_TOKEN }}' \
-            --header 'Content-Type: application/json' \
-            --data-raw '{
-                "tag_name": "v'"$VERSION"'",
-                "name": "module-ballerinax-java.jdbc-v'"$VERSION"'"
-            }'
-      - name: Post release PR
+      - name: Github Release and Release Sync PR
```

Comment:
```suggestion
      - name: GitHub Release and Release Sync PR
```

Reply:
`GitHub` is the correct capitalisation as per google. 😄

Reply:
Yeah. I am also suggesting to change `Github` to `GitHub` 😄

Reply:
Ah my bad.